love/src/components/MainTel/CameraHexapod/CameraHexapod.jsx

```diff
@@ -0,0 +1,234 @@
+
+    const commandableByDDS = hexapodCommandableByDDSStateMap[this.props.hexapodCommandableByDDS];
+
+    // controllerState
+    let controllerSubstate = '';
+    if (controllerState === 'Offline') {
+      controllerSubstate = hexapodControllerStateOfflineSubStateMap[this.props.hexapodControllerStateOfflineSubstate];
+    }
+    if (controllerState === 'Enabled') {
+      controllerSubstate = hexapodControllerStateEnabledSubstateMap[this.props.hexapodConstrollerStateEnabledSubstate];
+    } else {
+      controllerSubstate = 'Offline';
+    }
+
+    const inPosition = hexapodMTInPositionStateMap[this.props.hexapodInPosition];
+    const connected = hexapodConnectedStateMap[this.props.hexapodConnected];
+
+    return (
+      <div>
+        <div className={styles.hexapodContainer}>
+          <SummaryPanel className={styles.summaryPanel}>
+            <Title wide>Hexapod Status</Title>
+            <Label>Hexapod Status</Label>
+            <Value>
+              <StatusText status={hexapodStatusStatetoStyle[hexapodStatus.name]}>{hexapodStatus.name}</StatusText>
+            </Value>
+            <Label>Compensation</Label>
+            <Value>
+              <StatusText status={hexapodCompensationModeStatetoStyle[compensationStatus]}>
+                {compensationStatus}
+              </StatusText>
+            </Value>
+            <Label>ControllerSubstate</Label>
+            <Value>
+              <span className={styles.transformText}>{controllerSubstate}</span>
+            </Value>
+            <Label>Interlock state</Label>
+            <Value>
+              <span className={styles.transformText}>{interlockState}</span>
+            </Value>
+          </SummaryPanel>
+          <SummaryPanel className={styles.summaryPanel}>
+            <Title wide className={styles.titles}>
+              Readiness Summary
+            </Title>
+            <Label>Connected</Label>
+            <Value>
+              <StatusText status={hexapodConnectedStatetoStyle[connected]}>{connected}</StatusText>
+            </Value>
+            <Label>Commandable By DDS</Label>
+            <Value>
+              <StatusText status={hexapodCommandableByDDSStatetoStyle[commandableByDDS]}>{commandableByDDS}</StatusText>
+            </Value>
+            <Label>ControllerState</Label>
+            <Value>
+              <StatusText status={hexapodControllerStatetoStyle[controllerState]}>{controllerState}</StatusText>
+            </Value>
+            <Label>Hexapod in Position</Label>
+            <Value>
+              <StatusText status={hexapodMTInPositionStatetoStyle[inPosition]}>{inPosition}</StatusText>
+            </Value>
+          </SummaryPanel>
+        </div>
+        <div className={styles.divTables}>
+          <SimpleTable headers={this.HEADERS_HEXAPOD} data={newDataHexapod} />
+          <SimpleTable headers={this.HEADERS_STRUT} data={dataStrut} />
+        </div>
+      </div>
+    );
+  }
+}
+
+export default CameraHexapod;
```

Comment:
I'm missing `propTypes` and `defaultProps`, this is one of the standards that we started trying to acomplish. Take a look to what @fdelcampo did on the TMA Pull request: https://github.com/lsst-ts/LOVE-frontend/pull/352/files#diff-b48c5ff0fb0c99e8e54588a7d297b6c13d208cb149b514f96f3c9d528eb9bd1fR10